docs/guides/adapting-new-services.md

```diff
@@ -134,6 +134,7 @@ The structure of an entry in the [services.yml](../../config/services.yml) file
       - image/gif
     reprojection: true            # The service supports reprojection
   validate_variables: true        # Whether to validate the requested variables exist in the CMR. Defaults to true.
+  external_validation_url: http://example.com # Optional endpoint to be called to validate the user making a request
```

Comment:
I'm wondering if we should try to future proof it to allow for multiple validations and different types of validations. e.g.
```
validations:
  - url: !Env ${FOO_ENDPOINT}
  - docker: !Env ${FOO_IMAGE}
  - code: /app/validations/foo.ts
```
It's probably fine to keep it just a single URL for now and change it if needed in the future.

Reply:
Yeah, I think we will keep it simple for now and support those use cases as needed.